core/src/main/scala/stryker4s/Stryker4s.scala

```diff
@@ -8,13 +9,24 @@ import stryker4s.run.report.Reporter
 import stryker4s.run.threshold.{ScoreStatus, ThresholdChecker}
 
 class Stryker4s(fileCollector: SourceCollector, mutator: Mutator, runner: MutantRunner, reporter: Reporter)(
-    implicit config: Config) {
+    implicit config: Config) extends Logging {
 
   def run(): ScoreStatus = {
+    preRunLogs()
     val filesToMutate = fileCollector.collectFilesToMutate()
     val mutatedFiles = mutator.mutate(filesToMutate)
     val runResults = runner(mutatedFiles, fileCollector)
     reporter.report(runResults)
     ThresholdChecker.determineScoreStatus(runResults.mutationScore)
   }
+
+  private def preRunLogs(): Unit = {
+    // If JVM max memory is less than 2GB, log warning
+    if (!jvmHasEnoughMemory) {
+      warn("The JVM doesn't have a lot of memory assigned to it. It's wise to increase the maximum memory when running stryker.")
```

Comment:
`doesn't have enough memory assigned to it` makes it sound like it's impossible to run Stryker4s (not stryker) with the current settings. Logging what we think is ideal (other than in the command) is probably best too. Oh, and sbt is lowercase.